### ISSUE TITLE
Learning Rate and Num_of_Steps

### ISSUE THREAD

**Issue by eng100200** [liutinglt/CE2P#12]
@liutinglt In job_train.sh Learning rate is set 7e-3 while in train.py it is initialized as 7e-4. Similarly, in job-train.sh Num_Steps= 115000, while, in train.py it is 300000. I want to ask you which values i should used for training net?

**Comment by zed0630** [comment]
@eng100200 I think the parameters in job_train.sh is consistent with the original CE2P paper. The training set of LIP consists of about 30K images, which turns to about 750 iters/epoch with batch size=40. The training procedure contains about 150 epochs, which is 112500 iters in total. The initial learning rate is 0.007 in the paper, and you can find these information in Sec IV.B.
